README.md

```diff
@@ -60,6 +60,37 @@ The goal of writing an
 Python code to a Wasm module with exported functions that the host application
 can invoke. The first thing you should understand is creating an export.
 
+## Python Dependencies
+
+It is possible to add directories to the Python search path using the `PYTHONPATH` 
+environment variable. This means you should be able to install pure Python dependencies 
+into a virtual environment and add the library path to `PYTHONPATH` before executing
+`extism-py`, for example:
+
+Given code with an import from an external package
+
+```python
+import extism
+import toml # this is from pypi
+```
+
+You can add your dependencies to a virtual environment
+
+```bash
+$ python3 -m virtualenv ./deps
+$ source ./deps/bin/activate
+$ pip install toml
+```
+
+And then point `extism-py` to the dependencies
+
+```bash
+$ PYTHONPATH=./deps/lib/python3.12/site-packages extism-py -o a.wasm plugin.py
```

Comment:
is there a generally easy way to get `./deps/lib/python3.12/site-packages` from some CLI command we can recommend the user calls here, or is this just "`cd` around the locals `./deps` directory for where your `site-packages` is?

Reply:
Good call, I think this works:

```
 PYTHONPATH=$(python3 -c "import sys; print(sys.path[-1])") extism-py -o a.wasm plugin.py
```

Reply:
nice! thank you